motioneye/DOCS.md

```diff
@@ -124,6 +124,16 @@ within the motionEye UI.
 
 The bash shell command to be executed when the button is pressed.
 
+## motionEye Integration
+
+This add-on can optionally be used in conjuction with the Home Assistant
+[motionEye integration](https://www.home-assistant.io/integrations/motioneye/)
+to offer motionEye entities natively within Home Assistant.
+
+To configure the integration to use the add-on, simply use
+`http://localhost:28765` as the URL field in the integration configuration,
```

Comment:
Does the addon container name work?, i.e. http://a0d7b954-motioneye:28765

I know localhost works as it runs on the host network.

Reply:
Just checked: No, it doesn't.